src/main.rs

```diff
@@ -72,6 +76,7 @@ Options:
     -o --os <type>      Override the operating system [linux, osx, sunos]
     -u --update         Update the local cache
     -c --clear-cache    Clear the local cache
+    -p --pager    User `more` command to page output
```

Comment:
Can you align the help text with the other help text lines (using spaces)?

Also, I wonder if the pager behavior should be the default... Maybe a `--no-pager` option would be better? (CC @tamird)

Reply:
My fist implementation what following that but the thing is, since we are delegating the output to `more` the usability of it is weird because after print a command page, you have to hit  `Q` to close the pager.

![image](https://user-images.githubusercontent.com/6184100/44444445-2dd3a280-a5dd-11e8-9e27-96f35564de41.png)

You are stuck there until you press `q`

Reply:
Oh, shouldn't the pager only activate when the output is too large for the screen? That's what git does by default I think.

On the other hand the pager crate can't know that, so maybe we need to implement that ourselves...

Reply:
Maybe I didn't explain myself well. Pager only pages when the output is large, but when it actives, you have to hit `q` to get out of there.